src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INCREASE_FACTOR = 2;
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private Node<K, V>[] table;
+    private int size = 0;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= table.length * LOAD_FACTOR) {
+            resize();
+        }
+        int index = getIndexHash(key);
+        Node<K, V> newNode = new Node<>(key, value);
+        if (table[index] == null) {
+            table[index] = newNode;
+            size++;
+        } else {
+            Node<K, V> current = table[index];
+            while (current != null) {
+                if (isEqualKeys(current.key, key)) {
+                    current.value = value;
+                    return;
+                } else if (current.next == null) {
+                    current.next = newNode;
+                    size++;
+                    return;
+                } else {
+                    current = current.next;
+                }
```

Comment:
We do not need `else `after `return`
```suggestion
                } 
                if (current.next == null) {
                    current.next = newNode;
                    size++;
                    return;
                } 
                current = current.next;
```